62.CAD/fragment_shader.hlsl

```diff
@@ -11,9 +11,19 @@ void beginInvocationInterlockEXT();
 void endInvocationInterlockEXT();
 #endif
 
-// TODO[Lucas]: have a function for quadratic equation solving
-// Write a general one, and maybe another one that uses precomputed values, and move these to somewhere nice in our builtin hlsl shaders if we don't have one
-// See: https://github.com/erich666/GraphicsGems/blob/master/gems/Roots3And4.c
+float tForMajorCoordinate(float detRcp2, float bRcp) 
+{ 
+    float detSqrt = sqrt(detRcp2);
+    float2 roots = float2(-detSqrt,detSqrt)-bRcp;
+    // assert(roots.x == roots.y);
+    // assert(!isnan(roots.x));
+    return roots.x;
+}
+
+float evaluateBezier(float A, float B, float C, float t) 
```

Comment:
this will get removed. when you see the changes in, and you'll use the `Quadratic` struct
https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/53/files#diff-f1b9d90ab290c51d50a8e604e67bfd4dbc156ec2b1209d420313780d368069d9
https://github.com/Devsh-Graphics-Programming/Nabla/pull/527/files#diff-aeb209a11841655ba7cebc9d61b48153770cfa0c95c6d8367ab58c29e01a61db